test/qobjtoqbir.jl

```diff
@@ -71,22 +78,29 @@ YaoBlocks.@dumpload_fallback U2 U2
 YaoBlocks.@dumpload_fallback U3 U3
 
 function inst2qbir(inst)
-    n = maximum(x->maximum(x.qubits), inst) + 1
-    chain(n, map(inst) do x
-        name, locs = x.name, x.qubits .+ 1
-        nc = 0
-        while name[nc+1] == 'c' && nc<length(name)
-            nc += 1
-        end
-        if nc > 0
-            control(n, locs[1:nc], locs[nc+1:end]=>name_index(name[nc+1:end], x.params))
-        else
-            put(n, locs=>name_index(name, x.params))
-        end
-    end)
+    n = maximum(x -> maximum(x.qubits), inst) + 1
```

Comment:
we don't actually need to convert Qobj back to QBIR, since Qobj is for IBMQ client, but since you have done it, let's just include this too!